app/consumer-democracy/proposals_whitelisting.go

```diff
@@ -0,0 +1,68 @@
+package app
+
+import (
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	distrtypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
+	"github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	ibctransfertypes "github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
+)
+
+func IsProposalWhitelisted(content govtypes.Content) bool {
+
+	switch c := content.(type) {
+	case *proposal.ParameterChangeProposal:
+		return isParamProposalWhitelisted(c.Changes)
+
+	default:
+		return false
+	}
+
+}
+
+func isParamProposalWhitelisted(paramChanges []proposal.ParamChange) bool {
```

Comment:
Might be more clear if this method is changed to `isParamChangeWhitelisted`, since it's not referring to a proposal (correct?)

Reply:
yes, that's right